.github/workflows/tests.yml

```diff
@@ -477,6 +477,38 @@ jobs:
           installTics: true
           calc: ALL
 
+  trivy-repo:
+    name: Trivy vulnerability scanner - Repository
+    runs-on: ubuntu-22.04
+    permissions:
+      security-events: write # for uploading SARIF results to the security tab
+    if: ${{ ( github.event_name == 'workflow_dispatch' || github.event_name == 'schedule' ) }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: main
+
+      - name: Run Trivy vulnerability scanner in repo mode
+        uses: aquasecurity/trivy-action@master
+        with:
+          scan-type: "fs"
+          ignore-unfixed: true
```

Comment:
what does this do?

Reply:
Whoops, this should be removed. It ignores alerts that aren't fixed in newer versions of the vulnerable dependency/lib, e.g. a vulnerability that can't be fixed by updating our dependencies/libs.